### DISCUSSION TITLE
Add support for viewing text messages in external program

### DISCUSSION THREAD

**Discussion by artur-shaik** [d99kris/nchat#87] [Ideas]
Hello.

Sometimes I receive huge message which cannot be fully displayed in nchat. It would be cool If I can open this message, for example, in vim.

**Comment by d99kris** [comment]
Hi,
That sounds like a good idea, I will look into adding this functionality.

**Comment by d99kris** [comment]
Created issue #89 for tracking the implementation of this feature.

**Comment by d99kris** [comment]
The proposed functionality has been implemented in in 30dc5d6. If you update to latest nchat master it should be possible to select message (using key up/down) and then pressing `Alt-w` to view a message in an external program.

If you want to use `vim` you need to edit `~/.nchat/ui.conf` and set
```message_open_command=vim```
otherwise it uses `$PAGER` (or `less` if `$PAGER` is not set).

**Reply by artur-shaik** [reply]
Just tried it. Works perfect!

Thank you very much!
